Client/Frontend/Browser/ReaderModeBarView.swift

```diff
@@ -111,6 +111,7 @@ class ReaderModeBarView: UIView {
     }
 
     func SELtappedReadStatusButton(_ sender: UIButton!) {
+        UnifiedTelemetry.recordEvent(category: .action, method: .tap, object: .readingListItem, value: unread ? .markAsRead : .markAsUnread, extras: [ "from": "reader-mode-toolbar" ])
```

Comment:
Can you double-check this one? I was expecting the extras to have `["read": true/false]` because it would be clearer than a message with `tap | readingListItem | true/false`

Reply:
Or would this be `.readingListReadStatusButton` as the object?

Reply:
I changed this because I wanted to use the `extras` to help describe *how* we marked the item as read/unread since there's more than one way to do this.